tasks/generate_bdi_data.py

```diff
@@ -0,0 +1,302 @@
+import tempfile
+import shutil
+import os
+import math
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.tasks.bulkdata import LoadData
+from cumulusci.core.utils import ordered_yaml_load
+from cumulusci.utils import convert_to_snake_case
+from cumulusci.core.config import TaskConfig
+from datetime import date
+from datetime import timedelta
+from sqlalchemy import create_engine
+from sqlalchemy import Column
+from sqlalchemy import MetaData
+from sqlalchemy import Integer
+from sqlalchemy import Table
+from sqlalchemy import Unicode
+from sqlalchemy.ext.automap import automap_base
+from sqlalchemy.orm import create_session
+
+START_DATE = date(2019, 1, 1)
+
+
+class GenerateBDIData(BaseSalesforceApiTask):
+    def _run_task(self):
+        self.tempdir = tempfile.mkdtemp()
```

Comment:
There's a handy util you can use as a context manager that will take care of deleting the directory:

    from cumulusci.utils import temporary_dir
    with temporary_dir() as path:
        # it'll take care of changing to the directory within this block too

Reply:
> I did intend for feature/bdi-benchmark-test-data to be merged; that's why I made a PR for it.

Okay, thanks for the clarification. We'll need to coordinate what specifically should be merged then. Does the generate.py command line tool need to exist anymore? If so, we need to refactor to avoid the code duplication. Do they need an actual test_data.db file in the directories? etc.

I would advise against merging the parent branch until we've decided on these things.